code/modules/clothing/head/helmet.dm

```diff
@@ -568,3 +568,84 @@
 	fire = 50
 	acid = 50
 	wound = 30
+
+/obj/item/clothing/head/helmet/durability/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK, damage_type = BRUTE)
+	take_damage(1, BRUTE, 0, 0)
```

Comment:
doesn't giving something a throwforce value do this? or does that not work cause you can throw hats on people's heads.

Reply:
...? What this is meant to do, same as the other one, is to make the armour loose durability when the wearer is hit.

Reply:
ohhhhh ok sure